libs/types/src/cast_vote_records.ts

```diff
@@ -39,6 +40,12 @@ export interface CastVoteRecord
   readonly _scannerId: string;
 }
 
+export enum CastVoteRecordExportFileName {
```

Comment:
Nothing new in this file - just moving existing types from `libs/backend` into it (and shuffling some existing types around)